app/views/layouts/_header.html.erb

```diff
@@ -63,6 +63,7 @@
                     <li><%= link_to t('.edit_profile'), edit_user_path %></li>
                     <% if current_user.admin? %>
                       <li><%= link_to t('.admin'), '#admin_path', target: '_blank' %></li>
+                      <li><%= link_to t('.edit_frontpage'), edit_frontpage_path %></li>
```

Comment:
@Kerp1 had a theory that this was because of his recent PR that centered the contents on that row.